packages/components/src/components/as-histogram-widget/utils/draw.service.ts

```diff
@@ -293,9 +296,31 @@ function _delayFn(_d, i) {
   return i;
 }
 
-export function conditionalFormatter(value) {
-  if (value > 0 && value < 1) {
-    return decimalFormatter(value);
+function getFloatPrecision(value) {
+  const expValue = value.toString();
+  const expPos = expValue.indexOf('.');
+  return expPos > -1 ? expValue.slice(expPos + 1).length : 0;
+}
+
+function getDomainPrecision(domain) {
+  let domainPrecision = 0;
+  if (!(domain[0] instanceof Date)) {
```

Comment:
Just a silly question, but this is going to be always a Date instance? Checking just in case we don't have some `"2020-04-22T16:44:39.203Z"` strings.

Reply:
@neokore could you check that?

Reply:
Yes, domain is type Domain defined as follows: 
![image](https://user-images.githubusercontent.com/938057/80465539-883d0e80-893b-11ea-84f0-4c45deab338e.png)
Also, this domain is used in a `scaleLinear` object, that doesn't support text values (AFAIK).

Anyway, if you prefer we can check just if it's a number using `Number.isFinite` or just checking the type with `typeof`.

Reply:
Yes, I saw that type definition but we cannot perform type checking in values coming from VL at runtime, so that is why I was asking this :)

Reply:
Changed to check `Number.isFinite`, but according to MDN, this is not supported by IE and `tsconfig` is not set to `es2017` for components and `es5` for the bridge, so unless IE support were dropped from our side, I'll suggest setting components target to `es5` too.

Reply:
We're not supporting IE11 in VL also, so I would exclude it explicitly